### DISCUSSION TITLE
In App option for disabling or enabling NC or the transparency mode

### DISCUSSION THREAD

**Discussion by CheswickDEV** [steam3d/MagicPods-Windows#178] [Ideas]
I just got my AirPodsMax new and know that I can switch between transparency and NC mode via the button, but not to the normal "off" mode. It would be very nice if this could also be easily set via the app or at least displayed there, which mode is currently active.

**Comment by steam3d** [comment]
Hi. Apple highly hide how their AirPods work. It is not possible now.
